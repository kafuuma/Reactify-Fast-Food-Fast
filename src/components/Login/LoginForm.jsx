import React from 'react';
import {
  MDBContainer,
  MDBRow,
  MDBCol,
  MDBInput,
  MDBBtn,
  MDBCard,
  MDBCardBody
} from 'mdbreact';

const Login = ({ handleChange, handleSubmit }) => {
  const bgPink = { background: '#ff3300' };
  return (
    <MDBContainer>
      <MDBRow className="d-flex flex-center pt-5">
        <MDBCol md="6">
          <MDBCard>
            <MDBCardBody>
              <form onSubmit={handleSubmit}>
                <p className="h4 text-center py-4">Login</p>
                <div className="grey-text">
                  <MDBInput
                    name="email"
                    label="Your email"
                    icon="envelope"
                    group
                    type="email"
                    validate
                    error="wrong"
                    success="right"
                    onChange={handleChange}
                  />
                  <MDBInput
                    name="password"
                    label="Your password"
                    icon="lock"
                    group
                    type="password"
                    validate
                    onChange={handleChange}
                  />
                </div>
                <div className="text-center py-4 mt-3">
                  <MDBBtn id="bg" type="submit">
                    Login
                  </MDBBtn>
                </div>
              </form>
            </MDBCardBody>
          </MDBCard>
        </MDBCol>
      </MDBRow>
    </MDBContainer>
  );
};

export default Login;
