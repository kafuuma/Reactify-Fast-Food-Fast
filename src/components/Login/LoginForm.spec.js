import React from 'react';
import { shallow } from 'enzyme';
import LoginForm from './LoginForm';

describe('<LoginForm />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<LoginForm />);
  });

  it('matches snapshort', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
