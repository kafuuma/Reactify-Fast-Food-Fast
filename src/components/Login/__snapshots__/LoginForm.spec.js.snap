// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginForm /> matches snapshort 1`] = `ShallowWrapper {}`;
